src/toggle-switch/toggle-switch.scss

```diff
@@ -85,7 +85,7 @@
     aspect-ratio: 1 / 1;
     border-radius: 50%;
     transition: background-color $iui-speed-fast ease-out, opacity $iui-speed-fast ease-out;
-    z-index: 2;
+    z-index: 1;
```

Comment:
There is already `isolation: isolate` in this file, so these can be reverted.

Reply:
reverted

Reply:
left behind a typo
![image](https://user-images.githubusercontent.com/9084735/167707450-00fa069e-1978-4447-974b-c5b1607f0fcd.png)

Reply:
fixed